cmd/gaia/app/app.go

```diff
@@ -40,7 +40,7 @@ type GaiaApp struct {
 	*bam.BaseApp
 	cdc *codec.Codec
 
-	assertInvariantsBlockly bool
+	invariantsChecksPeriod uint
```

Comment:
```suggestion
	invCheckPeriod uint
```

Reply:
nit: "invariants check period" rolls off of the tongue better.

Reply:
LOL - fair enough